common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -472,8 +473,8 @@ def test_user_cannot_enroll_in_unknown_course_run_id(self, mock_get_course_runs)
         assert response.data['message'] == expected_message  # pylint: disable=no-member
         assert not CourseEnrollment.is_enrolled(self.user, fake_course_key)
 
-    @patch('lms.djangoapps.commerce.signals.refund_entitlement', return_value=[1])
-    @patch("entitlements.api.v1.views.get_course_runs_for_course")
+    @patch('entitlements.api.v1.views.refund_entitlement', return_value=True)
```

Comment:
Doesn't this live in the commerce utils?

Reply:
When you mock you have to mock from the file that it is being called and imported in to.